module-2/routes/users.ts

```diff
@@ -0,0 +1,84 @@
+import express from 'express';
+import { v4 as uuid } from 'uuid';
+import User from '../models/User';
+import { Users } from '../db';
+import schema from '../schema';
+import { filterNotDeleted, getAutoSuggestUsers } from '../utils';
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  res.send([...Users.values()].filter(filterNotDeleted));
+});
+
+router.get('/all', (req, res) => {
+  res.send([...Users.values()]);
+});
+
+const DEFAULT_LIMIT = 3;
```

Comment:
Consider creating a separate file for such constants at the app.js level or in a designated folder such as "core" - it might store all the common reusable declarations and types.

Reply:
Moved shared constants to `consts.ts`, thanks